onnxruntime/contrib_ops/cuda/bert/packed_multihead_attention.cc

```diff
@@ -229,17 +230,32 @@ Status PackedMultiHeadAttention<T>::ComputeInternal(OpKernelContext* context) co
   auto& device_prop = this->GetDeviceProp();
   MHARunner* fused_runner = this->GetFusedRunner(device_prop, parameters);
 
-  bool use_memory_efficient_attention = false;
+  bool use_flash_attention = false;
 
 #if USE_FLASH_ATTENTION
+  if (nullptr == fused_runner) {
```

Comment:
flash attention v2 shall have higher priority than fused_runner.  When v2 is avaiable, we shall set fused_runer to be nullptr.